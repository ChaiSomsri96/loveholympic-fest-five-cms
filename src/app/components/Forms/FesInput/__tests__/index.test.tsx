import * as React from 'react';
import { render } from '@testing-library/react';

import { FesInput } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<FesInput  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<FesInput />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
