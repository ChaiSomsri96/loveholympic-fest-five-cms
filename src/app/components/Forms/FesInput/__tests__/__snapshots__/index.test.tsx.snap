// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FesInput  /> should match snapshot 1`] = `
.c0 {
  background-color: #455167;
  color: #fff;
  padding-left: 2.5rem;
}

.c0.input {
  background-color: #455167 !important;
}

.c0.form-control:focus {
  background-color: #455167 !important;
  color: #fff;
}

.c0::-webkit-input-placeholder {
  color: #fff;
  text-transform: uppercase;
  font-weight: 50;
  font-size: 14px;
  opacity: 50%;
}

<div
  class=""
>
  <input
    class="c0 form-control"
    type="text"
    value=""
  />
</div>
`;
